README.md

```diff
@@ -21,3 +21,25 @@ You can install this game on your own subreddit if you would like to give it a t
 ## Source Code
 
 The [code for HotAndCold is open source](https://github.com/reddit/devvit-HotAndCold)! Feel free to suggest improvements, fork, and submit PRs of new features and improvements.
+
+## Development
```

Comment:
We should also update the existing dev Devvit config so it follow the same pattern

Reply:
Can we also protect the prod devvit config by renaming it so it isn't the default? Otherwise, it will be easy to accidentally modify the prod app.